jetty9/pom.xml

```diff
@@ -27,12 +27,6 @@
       <version>${project.version}</version>
       <type>tar.gz</type>
     </dependency>
-    <dependency>
```

Comment:
@ludoch The Host header is required for HTTP/1.1 and commonly sent with HTTP/1.0.   The X-AppEngine-Https header is used elsewhere in the code to determine if the request is secure.  You can see both headers in a [dump](https://jetty9-work.appspot.com/dump/info)